bimcollab_zoom/com.github.wycomco.bimcollab_zoom.munki.recipe

```diff
@@ -8,37 +8,42 @@
 	<string>com.github.wycomco.bimcollab_zoom.munki</string>
 	<key>Input</key>
 	<dict>
+		<key>APPLICATIONS_PATH</key>
+		<string>/Applications</string>
 		<key>MUNKI_REPO_SUBDIR</key>
 		<string>apps/%NAME%</string>
 		<key>NAME</key>
-		<string>bimcollab_zoom</string>
-		<key>NICE_NAME</key>
-		<string>BIMcollab ZOOM</string>
-		<key>PKGPAYLOAD</key>
+		<string>BIMCollabZoom</string>
+		<key>PKGPAYLOAD_DIR</key>
 		<string>pkgpayload</string>
-		<key>PKGROOT</key>
+		<key>PKGROOT_DIR</key>
 		<string>pkgroot</string>
 		<key>RELATIVE_APP_PATH</key>
-		<string>/BIMcollab ZOOM/BIMcollab ZOOM.app</string>
+		<string>%APPLICATIONS_PATH%/BIMcollab ZOOM/BIMcollab ZOOM.app</string>
 		<key>pkginfo</key>
 		<dict>
 			<key>catalogs</key>
 			<array>
 				<string>testing</string>
 			</array>
+			<key>category</key>
+			<string>Graphics and Design</string>
```

Comment:
Is this a common category? I know, that we are using ist quite often, but how about the AutoPKG community?